src/spotbugs/spotbugs-excludes.xml

```diff
@@ -0,0 +1,14 @@
+<FindBugsFilter>
+  <!-- disable all findsecbugs findings -->
+<!--    <Match>
```

Comment:
If a plugin / component really couldn't handle findsecbugs or had an urgent need. This really isn't a good idea.